src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1418,6 +1418,122 @@ def aks_upgrade(cmd, client, resource_group_name, name, kubernetes_version, no_w
     return sdk_no_wait(no_wait, client.create_or_update, resource_group_name, name, instance)
 
 
+def aks_use_devconnect(cmd, client, cluster_name, resource_group_name, space_name='default'):  # pylint: disable=line-too-long
+    """
+    Use Azure Dev Connect with a managed Kubernetes cluster.
+
+    :param cluster_name: Name of the target AKS cluster.
+    :type cluster_name: String
+    :param resource_group_name: Name of the target AKS cluster's resource group.
+    :type resource_group_name: String
+    :param space_name: The isolated space in the cluster to develop in.
+    :type space_name: String
+    """
+
+    vsce_tool = 'Visual Studio Connected Development Services'
+    vsce_cli = 'vsce'
+    install_vsce = False
+    system = platform.system()
+    if system == 'Linux':
+        # Linux
+        raise CLIError('Linux is not supported yet.')
+    elif system == 'Windows':
+        # Windows
+        # Dev Connect Install Path (WinX)
+        vsce_cli = "C:\\Program Files\\Microsoft SDKs\\Azure\\Visual Studio Connected Environment CLI\\vsce.exe"
+        setup_file = 'vsce-winx-setup.exe'
+        setup_url = "https://aka.ms/get-vsce-windows-az"
```

Comment:
The .exe does not appear signed. I just tried downloading it.

Reply:
This is temporary change, we are pointing to dev url currently. We will update the aka.ms link to point to prod url, then it would be signed.